src/lib/definitions.js

```diff
@@ -985,6 +986,7 @@ function getFHPaddingEntries(index)
         'P':
         [
             define('String.fromCharCode("80")'),
+            define('(RP_0_S + Function("return async function () {}")()())[11]', ASYNC_FUNCTION),
```

Comment:
Whitespaces inside a string are encoded as they are, so it's better to omit them when possible.

```suggestion
            define('(RP_0_S + Function("return async function(){}")()())[11]', ASYNC_FUNCTION),
```